### ISSUE TITLE
Debug info is lacking or nonexistant

### ISSUE THREAD

**Issue by asacavanagh** [PEXPlugins/PermissionsEx#2433]
Hello

I've recently just reset my server and have been having some issues with permissions issues as you do when you start fresh. In the past I've relied heavily on the PeX debug information, however since using Spigot 1.9.4 (Unsure when exactly the issue itself would have started as it worked okay previously on 1.9.3 and below), PeX seems to refuse to display any information at all when using the command "/pex user <name> toggle debug". When you do the command, console output that the debug mode for the user has been toggled on, however nothing at all appears to come up afterward.

Previously I'd be seeing permissions checks for worldguard if I were to move or permissions checks for sending messages to chat and so on, now nothing at all appears to show up, even in the latest.log of spigot. If I toggle it globally using 'pex toggle debug', it does display very.. very.. limited permissions checks (that being from only 1 or 2 plugins and nothing else at all).

Spigot version: 
CraftBukkit version git-Spigot-8a048fe-5ff377a (MC: 1.9.4) 
(Implementing API version 1.9.4-R0.1-SNAPSHOT)

My PeX config file: 
http://pastebin.com/dqTGwRdr

Console output: 
http://pastebin.com/c94h8pEF
Note - In the time I had debug enabled for myself, I talked in chat, moved around and broke a few blocks (none of which was checked for permissions, according to the debug).

I've probably missed something simple in my configuration somewhere but it's becoming an issue now as players are finding more problems that I could easily solve with the aid of PeX debug.

Any help is appreciated, thanks!
Asa

**Comment by zml2008** [comment]
Unfortunately user-specific debug toggling had to be removed because of performance issues. You'll have to use global debug mode for now.

**Comment by asacavanagh** [comment]
If this were a solution, I wouldn't of created this issue on here. However if you actually look at what I posted I say:

"If I toggle it globally using 'pex toggle debug', it does display very.. very.. limited permissions checks (that being from only 1 or 2 plugins and nothing else at all)."

I did forget to add a snippet of the output from global debug, so here it is: http://pastebin.com/jTcP774F

I've used the debug information generated from PeX in the past to fix a large amount of issues, so to have it only output minimal permissions checks and not every single one it's actually doing is absolutely useless. From what I can tell important checks like worldguard and essentials for example aren't showing up at all or if they are they're not showing up very often or not showing up any permissions that matter. Was the global debug output scaled back as well for 'performance issues'?
